frontend/config/dev.webpack.config.js

```diff
@@ -65,7 +65,9 @@ const webpackProxy = {
   customProxy: [
     {
       context: (path) => path.includes('/api/k8s/registration'),
-      target: 'https://registration-service-toolchain-host-operator.apps.appstudio-stage.x99m.p1.openshiftapps.com',
+      target: process.env.STAGE
+        ? 'https://registration-service-toolchain-host-operator.apps.stone-stg-host1.hjvn.p1.openshiftapps.com/'
```

Comment:
@karelhala does the trailing slash matter here?

Reply:
Good point!